test/manual/sstable_scan_footprint_test.cc

```diff
@@ -266,15 +266,15 @@ void test_main_thread(cql_test_env& env) {
     stats_collector sc(sem, stats_collector_params);
     try {
         auto _ = sc.collect();
-        memory::set_heap_profiling_enabled(true);
+        memory::set_heap_profiling_sampling_rate(100);
```

Comment:
@michoecho @avikivity  a breaking change sneaked in anyway. shall we add a follow-up change to bring this API back, like implemented with a reasonable default sampling rate ?

Reply:
I think we can let it slip, if it hits anyone else (unlikely), it's a not in any important place.

Reply:
ack. thanks.